packages/plugin-core/src/constants.ts

```diff
@@ -872,15 +897,32 @@ export const DENDRON_COMMANDS: { [key: string]: CommandEntry } = {
     title: `${CMD_PREFIX}Dev: Diagnostics Report`,
     when: DendronContext.PLUGIN_ACTIVE,
   },
+  /**
+   * This name is a bit of a misnomer - this actually launches the welcome
+   * screen, which has a button that will launch the tutorial when clicked.
+   */
   LAUNCH_TUTORIAL: {
     key: "dendron.launchTutorial",
     title: `${CMD_PREFIX} Launch Tutorial`,
   },
+  /**
+   * This command actually launches the tutorial workspace
+   */
+  LAUNCH_TUTORIAL_WORKSPACE: {
+    key: "dendron.launchTutorialWorkspace",
+    title: `${CMD_PREFIX} Launch Tutorial Workspace`,
+    when: "false",
+  },
   OPEN_BACKUP: {
     key: "dendron.openBackup",
     title: `${CMD_PREFIX} Open Backup`,
     when: DendronContext.PLUGIN_ACTIVE,
   },
+  INSTRUMENTED_WRAPPER_COMMAND: {
```

Comment:
can you add a comment to what this means?

Reply:
The comment is in `InstrumentedWrapperCommand.ts`